gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -373,11 +372,9 @@ def calculate_optimal_partition_size(total_base_pairs):
           partition_size_round_up * (_MAX_BQ_NUM_PARTITIONS - 1))
 
 
-def compose_table_name(base_name, suffix, is_sample=False):
+def compose_table_name(base_name, suffix):
   # type: (str, List[str]) -> str
```

Comment:
Is the second argument type `List[str]`? singular name does not suggest that.

Reply:
Nice catch, modified.